pkg/engine/renderer/parser.go

```diff
@@ -22,7 +22,7 @@ func YamlToObject(yaml string) (objs []runtime.Object, err error) {
 		}
 
 		decode := scheme.Codecs.UniversalDeserializer().Decode
-		obj, _, e := decode([]byte(f), nil, nil)
+		obj, gvk, e := decode([]byte(f), nil, nil)
```

Comment:
🤷‍♀ decode returns runtime object without GVK set and then GVK in separate value, I asked about this in apimachinery slack, did not get answer I would understand :) I guess it had some reason some time ago :D